### ISSUE TITLE
df.global.world.items.other.ANY_COOKABLE may have misleading name

### ISSUE THREAD

**Issue by PatrikLundell** [DFHack/df-structures#303]
I was trying to locate items that could be cooked, and started to look at what was in the vector mentioned above. However, in my 0.44.12 fortress there are a lot of things that would be cooked only as the result of losing a bet, such as artifact item_toolst with written contents, item_flaskst, item_cagest, item_bucketst, , item_barrelst, and item_boxst (in addition to a lot of things that can probably be cooked, such as e.g. seeds, eggs, drinks, and misc powders ["probably" means I haven't looked deeper to verify the particular seeds are of a cookable kind, etc.]).
I have no theory of what the vector is supposed to contain, though, as all the flasks in my fortress are vials (so it can't be organic).

**Comment by lethosor** [comment]
If it didn't have food in it, my guess would be some missing vectors, like #235. However, that usually breaks other tools that deal with items in various ways, and we haven't had issues with that in 0.44.12-r1 that I know of. However, just to be sure, do any of the nearby vectors in world.items.other have an appropriate name for this group of items?

**Comment by PatrikLundell** [comment]
No, the vector has food in it, but it also has non food in it. Most of the non food item types are containers, but "books" obviously aren't. The nearby vectors seem to contain what they're supposed to, given their names, while none of the generic ones seem to contain cookable items exclusively. ANY_GENERIC84 (the next one) seems to contain boxes only (I haven't checked everything), and so can't contain all cookable items, even if the boxes contain things following a common theme [which I haven't checked if they do]).

Thus, it seems the vector is misidentified, and is intended for some other purpose.
